### ISSUE TITLE
"Request failed with status code: 500" causes the app to hang 

### ISSUE THREAD

**Issue by MeijeSibbel** [GooBox/goobox-sync-storj#9]
```
Checking if local Goobox folder exists... yes
Checking if cloud Goobox bucket exists... yes
Checking Storj Bridge for changes...
Sleeping for 1 minute...
Checking Storj Bridge for changes...
Request failed with status code: 500
```

After this it just stays there, it doesn't ignore the issue and move on.

**Comment by kaloyan-raev** [comment]
Yes. I see this error tons of time today. It seems that the Bridge has a bad date.

But, for sure, the app should not hang when this happens.

**Comment by kaloyan-raev** [comment]
This was partially fixed in version 0.0.3. PR https://github.com/GooBox/goobox-sync-storj/pull/17 resolves the rest.
